Makefile

```diff
@@ -512,7 +512,7 @@ COMMON_SRC = \
             fc/config.c \
             fc/fc_tasks.c \
             fc/fc_msp.c \
-            fc/mw.c \
+            fc/fc_main.c \
```

Comment:
Well, we could call it `finno_ugric.c` , Finno-Ugric being the group where you put the leftover things once you have done your main classification. :-)